src/language-js/print/assignment.js

```diff
@@ -19,6 +19,8 @@ import {
   isLoneShortArgument,
   isObjectProperty,
   createTypeCheckFunction,
+  isUnionType,
+  isIntersectionType,
```

Comment:
WDYT of adding https://github.com/lydell/eslint-plugin-simple-import-sort to the codebase? This will automatically make import lists alphabetically ordered. The diffs will be more predictable and specific import names will be easier to find. These lists are currently ordered based on the introduction time, which makes sense historically, but not so much when you look at the code snapshot.

Reply:
Good idea.

Reply:
Do you want to do it? I can craft a quick PR if you’re busy with other stuff.

Reply:
It will be great, if you can work on it.

Reply:
→ https://github.com/prettier/prettier/pull/15817